tests/task_t_1.spec.js

```diff
@@ -10,4 +10,41 @@ test('duck duck go T 1', async ({ page }) => {
   await page.goto('https://duckduckgo.com/');
   const logo = page.locator('#logo_homepage_link');
   await expect(logo).toBeVisible;
+});
+
+test('T2: Check that search results contain expected text', async ({ page }) => {
+  await page.goto('https://duckduckgo.com/');
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('Test');
+  await page.click('#search_button_homepage');
+  const searchResults = page.locator('#links').toBeVisible;
+  const firstResult = await page.textContent('#r1-0');
+  await expect(firstResult).toContain('Test');
+});
+
+test('T3: Check that Cheat sheet for MSWord is displayed', async ({ page }) => {
+  await page.goto('https://duckduckgo.com/');
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('Microsoft word cheat sheet');
+  await page.click('#search_button_homepage');
+  const firstResult = await page.textContent('.c-base__title');
+  await expect(firstResult).toContain('Microsoft Word 2010');
+  await page.isVisible('span.chomp--link__mr');
+  expect(await page.click('span.chomp--link__mr')).click;
+
+  expect(await page.isVisible('h6.cheatsheet__title:has-text("Formatting")'))
```

Comment:
This should work fine, but I would suggest validating text separately, not inside of a selector for better code readability ant maintainability, e.g. `const textContent = await page.textContent('h6.cheatsheet__title')`